src/sentry/receivers/outbox/control.py

```diff
@@ -68,22 +69,30 @@ def process_async_webhooks(payload: Mapping[str, Any], region_name: str, **kwds:
 
     if SiloMode.get_current_mode() == SiloMode.CONTROL:
         # By default, these clients will raise errors on non-20x response codes
-        response = RegionSiloClient(region=region).request(
-            method=webhook_payload.method,
-            path=webhook_payload.path,
-            headers=webhook_payload.headers,
-            # We need to send the body as raw bytes to avoid interfering with webhook signatures
-            data=webhook_payload.body,
-            json=False,
-        )
-        logger.info(
-            "webhook_proxy.complete",
-            extra={
-                "status": response.status_code,
-                "request_path": webhook_payload.path,
-                "request_method": webhook_payload.method,
-            },
-        )
+        try:
+            response = RegionSiloClient(region=region).request(
+                method=webhook_payload.method,
+                path=webhook_payload.path,
+                headers=webhook_payload.headers,
+                # We need to send the body as raw bytes to avoid interfering with webhook signatures
+                data=webhook_payload.body,
+                json=False,
+            )
+            logger.info(
+                "webhook_proxy.complete",
+                extra={
+                    "status": getattr(
+                        response, "status_code", 204
+                    ),  # Request returns empty dict instead of a response object when the code is a 204
+                    "request_path": webhook_payload.path,
+                    "request_method": webhook_payload.method,
+                },
+            )
+        except ApiError as e:
+            if e.code == 409:
```

Comment:
Would it be worth turning this into a proper ApiError subtype? We have a couple, such as `ApiUnauthorized`

Reply:
I think having a separate type makes sense as we have application logic tied to conflicts. We have a few other integration webhook handlers that will need to be normalized to use 409. Currently jira and a few others are using 400 for un-processable hook payloads.